### PR TITLE
Fixed Stream Unite entry build.

### PR DESCRIPTION
**Pull request by cederom**
ConnectionFX3Entry(void) does not accept parameters.

Signed-off-by: Tomasz 'CeDeROM' CEDRO <tomek@cedro.info>

### CODE REVIEW

**Review by TR3FFN1X — APPROVED** [review]

**Review by TR3FFN1X — APPROVED** [review]

**Comment by cederom** [comment]
TANKS =)

**Comment by 9600** [comment]
@cederom these looks like spam "approvals"...

**Comment by cederom** [comment]
hmm maybe permissions to the repo needs an update? i was able to edit wiki. i guess edit, approve, commit should require "member" or "manager" permissions? :-)

**Comment by 9600** [comment]
Looks like anyone can approve if you configure for reviewers, but this doesn't mean anything if those approving have no write/commit access.

We don't use the wiki functionality on GitHub and I've just disabled it on this repo. Our wiki is at:

https://wiki.myriadrf.org/Welcome

If you'd like edit this I would need to create an account for you.

**Comment by cederom** [comment]
We did a scratch pad of porting LimeSuite to FreeBSD here on wiki github because its closest point to the source code and patches / pull requests :-) Port is ready but there is still some reset stuff to be solved (either on LimeSuite or OS side). If possible and preferred we can edit wiki on MyriadRF :-) Please let us know what you prefer @9600 :-)

Also if you guys need technical support in your projects I am open for B2B (EU TAX ready) contracts :-)

**Comment by rjonaitis** [comment]
"STREAM+UNITE" name string is needed to differentiate from the base FX3 connection, as the name string is used to index connections in the array.
https://github.com/myriadrf/LimeSuite/blob/master/src/ConnectionRegistry/ConnectionRegistry.cpp#L104-L109

Fixed in: a45e482dad28508d8787e0fdc5

**Comment by cederom** [comment]
Hmm, please verify again, as the `ConnectionFX3Entry(void)` function will not accept a string parameter and thus the build is broken.. maybe different function should be used here instead? :-)

**Comment by rjonaitis** [comment]
I've fixed it by adding another constructor which does take a parameter.

**Comment by cederom** [comment]
Awsome! Thank you! :-)

**Pull request closed without merging**
